RELEASES.md

```diff
@@ -1,3 +1,7 @@
+# 0.8.1
+### Change
+- Issue #61 - Wrong bls dependency
```

Comment:
Can you explain a bit instead of linking the issue?

Reply:
Fixed